SLAM/EKFSLAM/ekf_slam.py

```diff
@@ -94,15 +94,11 @@ def observation(xTrue, xd, u, RFID):
 
 
 def motion_model(x, u):
-    F = np.array([[1.0, 0, 0],
-                  [0, 1.0, 0],
-                  [0, 0, 1.0]])
-
     B = np.array([[DT * math.cos(x[2, 0]), 0],
                   [DT * math.sin(x[2, 0]), 0],
                   [0.0, DT]])
 
-    x = (F @ x) + (B @ u)
```

Comment:
Could you please keep (F@x)?. Because, this repo's aim is for learning algorithm. So the code should be same as algorithm. In this simulation case, it is possible to simplify the code, but in other simulation, it might be impossible.